pkg/resolver/service/service.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package service
+
+import (
+	"github.com/ethersphere/bee/pkg/logging"
+	"github.com/ethersphere/bee/pkg/resolver"
+	"github.com/ethersphere/bee/pkg/resolver/client/ens"
+)
+
+// InitMultiResolver will create a new MultiResolver, create the appropriate
+// resolvers, push them to the resolver chains and attempt to connect.
+func InitMultiResolver(logger logging.Logger, cfgs []*resolver.ConnectionConfig) *resolver.MultiResolver {
+	if len(cfgs) > 0 {
+		logger.Info("no name resolution service provided")
+	}
+
+	logger.Debug("connecting to name resolution services")
```

Comment:
This log line does not provide much information. Maybe to add a log line for every connection that is made with some context about that connection (address, etc...).